packages/sandcastle/src/Gallery/loadFromUrl.ts

```diff
@@ -0,0 +1,103 @@
+import { type GalleryItem } from "./GalleryItemStore.ts";
+import { decodeBase64Data } from "../Helpers.ts";
+import { getBaseUrl } from "../util/getBaseUrl.ts";
+
+const loadGist = async (gist: string) => {
```

Comment:
Can you please add back some comment that this is purely for legacy support?

Edit: I now see the comment below where this is called. Maybe should be in both places but up to you.